site/zenodo_rdm/migrator/extract.py

```diff
@@ -23,8 +23,6 @@ def run(self):
         """Yield one element at a time."""
         with open(self.filepath, "r") as reader:
             for line in reader:
-                # TODO JSONl file should be cleaned up before loading
-                line = line.strip().replace("\\\\", "\\")
                 yield json.loads(line)
```

Comment:
Does it still need to be loaded or can we use `jsonlines.read` now?

Reply:
Are we sure we don't need still to use `.strip()`?

Reply:
> Does it still need to be loaded or can we use `jsonlines.read` now?

I think that's an [additional dependency](https://jsonlines.readthedocs.io/en/latest/), not sure if/where it's defined, and looks like there aren't many of its features we'd be using.

> Are we sure we don't need still to use `.strip()`?

Looks like it's not needed for `json`:

```python
In [1]: import json
In [2]: json.loads(" {} ")
Out[2]: {}
```